theano/gof/op.py

```diff
@@ -780,6 +780,10 @@ class Op(utils.object2, PureOp, CLinkerOp):
 
     """
     def __new__(cls, *args, **kwargs):
+        # TODO: Ensuring all the Ops created henceforth has __props__
+        #       This helps to ease the process of caching by making use
+        #       of the _props_dict() instead of inferring the parameters
+        #       passed to the class's constructor.
```

Comment:
Remove this comment. I don't think for now that it would be a good idea to make \__props__ mandatory. I don't see that coming in the futur.